src/grpc/GrpcServiceError.js

```diff
@@ -55,10 +55,19 @@ export default class GrpcServiceError extends Error {
         if (obj.code != null && obj.details != null) {
             const status = GrpcStatus._fromValue(obj.code);
             const err = new GrpcServiceError(status);
-            err.message = obj.details;
+            err.stack +=
+                "\nCaused by: " + (obj.stack ? obj.stack.toString() : "");
```

Comment:
For keeping consistency I would suggest to use template literals here 
```suggestion
            err.stack += `\nCaused by: ${obj.stack ? obj.stack.toString() : ""}`;
```